tests/unit_tests/c_wrappers/res/enkf/test_field_export.py

```diff
@@ -60,3 +64,57 @@ def test_field_export_many(snake_oil_field_example):
     assert os.path.isfile("export/with/path/PERMX_0.grdecl")
     assert os.path.isfile("export/with/path/PERMX_2.grdecl")
     assert os.path.isfile("export/with/path/PERMX_4.grdecl")
+
+
+def test_field_init_file_not_readable(copy_case):
+    copy_case("snake_oil_field")
```

Comment:
Likely the test could run faster if you used a smaller example. snake_oil_field is quite extensive.

Reply:
indeed! i'm gonna see if i can fix : )

i also realize now that you're right, both examples don't fail at startup, but after running, so what I wrote in the OBS is not actually relevant here

Reply:
OK looking at it more carefully, the first test, the one without `FORWARD_INIT:True`, fails while trying to sample priors, I believe at the start of running simulations, and that one could fail faster (experience when running in command line: the exception gets logged after like 3 seconds, but the main process dies only after one minute)

Reply:
made it faster!